src/data/data.json

```diff
@@ -14,8 +14,8 @@
         },
         {
           "name": "Leia Organa",
-          "cost": 90,
-          "rank": "commander",
+          "point_cost": 90,
```

Comment:
Do you intend on having other ways of "paying" other than points? I thought cost was fine.

Reply:
I wanted to be explicit, so it was obvious what it mapped too? I dunno ahaha